action.yml

```diff
@@ -43,12 +30,36 @@ inputs:
       HINT: use $ {{ secrets.DOCKERHUB_TOKEN }} (no space after $)
     default: ""
     required: false
+  # OPTIONAL: ghcr.io
+  ghcr_token:
```

Comment:
I'm unsure how I feel about explicitly listing dockerhub and ghcr credentials, vs using a generic username/password and documenting what to fill in for various repos.

Reply:
Addressed in [WIPACrepo/wipac-dev-workflows@`7ae3f73` (#1)](https://github.com/WIPACrepo/wipac-dev-workflows/pull/1/commits/7ae3f7322355d125a99f08f5b621878246a8c3cd)